src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import FooterLogo from "../../assets/netflix-logo.png";
+import "./Footer.scss";
+
+export default function Footer() {
+    return(
+        <footer className="footer">
+            <div className="container">
+                <div className="logo">
```

Comment:
Hi Natalia,
it would be great to extract logo's markup in a separate component and reuse it in the Header.

Reply:
Hi @inuctio,
Thanks, fixed.